lib/krikri/registry.rb

```diff
@@ -0,0 +1,45 @@
+module Krikri
+
+  ##
+  # Provide a registry of defined items that can be looked up by a token
```

Comment:
See also this StackOverflow answer: [When to use symbols versus strings in Ruby?](http://stackoverflow.com/a/16621092/1277323)